example/main.dart

```diff
@@ -2,6 +2,7 @@ import 'package:flutter/material.dart';
 import 'package:matrix/matrix.dart';
 import 'package:path_provider/path_provider.dart';
 import 'package:provider/provider.dart';
+import 'package:sqflite/sqflite.dart';
```

Comment:
I think we should import this as sqlite, so that the openDatabase becomes `sqlite.openDatabase`. Otherwise users might confuse it and assume it is a function from our SDK.

Reply:
Done :)